tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/prj.conf

```diff
@@ -1,2 +1,3 @@
 CONFIG_CPLUSPLUS=y
 CONFIG_NEWLIB_LIBC=y
+CONFIG_ETH_LITEETH=n
```

Comment:
What is the intended use of these line? Does it fix the problem with writing to ethernet we had earlier?

Reply:
This line is disabling the ethernet support on the board. Yes, it was related to the issue we had earlier.